crates/bdk/tests/wallet.rs

```diff
@@ -753,7 +765,7 @@ fn test_create_tx_drain_to_dust_amount() {
     builder
         .drain_to(addr.script_pubkey())
         .drain_wallet()
-        .fee_rate(FeeRate::from_sat_per_vb(453.0));
+        .fee_rate(FeeRate::from_sat_per_vb_unchecked(499));
```

Comment:
Why changing from 453 to 499?

Reply:
Admittedly, this was a hack to make the test pass, and `499` was the first value chosen because it gives the impression of a "very high fee rate." The original `453` must have been right on the edge of passing. Evidently `454` would make this pass as well.

Reply:
I can confirm that 454 passes the tests, can we use that?

```suggestion
        .fee_rate(FeeRate::from_sat_per_vb_unchecked(454));
```